vcd/datasource_vcd_resource_list.go

```diff
@@ -148,7 +148,7 @@ func getPvdcList(d *schema.ResourceData, meta interface{}) (list []string, err e
 func getVdcGroups(d *schema.ResourceData, meta interface{}) (list []string, err error) {
 	client := meta.(*VCDClient)
 
-	org, err := client.GetAdminOrgByName(d.Get("org").(string))
+	org, err := client.GetAdminOrg(firstNonEmpty(d.Get("org").(string), d.Get("parent").(string)))
```

Comment:
Supporting both fields for
the same purpose will cause some issues down the road (like ambiguity). What about just throwing a meaningful error if 'parent' is used in a wrong context?

Reply:
The documentation still mentions `"org"` as the necessary field.
Using `"parent"` could be a honest mistake that is difficult to diagnose when happens (it fooled me as well in a case). This change makes the error disappear without side effects.

Anyway, the main point of the fix is making sure that the `"org"` from the provider is being used when the one in the data source is missing. That was not always the case, and the fix for `"parent"` was a cheap addition.